src/vs/code/electron-main/window.ts

```diff
@@ -334,6 +335,22 @@ export class VSCodeWindow {
 		return this._readyState;
 	}
 
+	private registerNavigationListenerOn(command: string, back: string, forward: string) {
+		this._win.on(command, (e, cmd) => {
+			if (this.readyState !== ReadyState.READY) {
+				return; // window must be ready
+			}
+
+			if (cmd === back) {
+				this.send('vscode:runAction', 'workbench.action.navigateBack');
+			} else if (cmd === forward) {
+				this.send('vscode:runAction', 'workbench.action.navigateForward');
+			} else {
+				console.warn('[electron command]: did not handle: ', command);
```

Comment:
But we are calling this method, no? Why do we expect to print a warning? This is not API anyone else is using?